index.html

```diff
@@ -45,36 +56,108 @@
         padding: 20px;
         width: 100%;
       }
+
+      #dice-number,
+      #player-number,
+      #game-mode {
+        width: 60%;
+      }
+
+      #game-mode {
+        font-size: 21px;
+        line-height: 33px;
+        margin: 0 0 23px 0;
+        padding: 0 9px;
+        width: 100%;
+        width: 150px;
+        height: 37px;
+        text-align: center;
+        font-size: medium;
+      }
+
+      #gameRules {
+        width: 100%;
+        text-align: center;
+      }
+
+      #user-input {
+        display: flex;
+        flex-direction: row;
+        flex: auto;
+      }
+
+      .inputInfo {
+        text-align: center;
+        width: 250px;
+      }
+
+      #game-output {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        flex: auto;
```

Comment:
very good attempt at playing around with css esp flexbox! definitely gonna make module 1 of bootcamp smoother for you